tests/spread-function-call/spread-function-call.es5

```diff
@@ -0,0 +1,47 @@
+var list = [1, 2, 3];
+
+Array.prototype.expand = false;
+
+function __(list) {
+  list.expand = true;
+  return list;
+}
+
+// wrap the function to be called, so that the arguments are parsed first.
+function wrap(func) {
```

Comment:
In cases that this kind of behaviour was expected (using a list as an argument, but expecting more than one argument) i'd use .apply, but to simulate the exact same behaviour as the list speading in ES6 this is what I came up with.